### ISSUE TITLE
Media Source files not [x] CLOSE when inactive

### ISSUE THREAD

**Issue by MACKerMD** [obsproject/obs-studio#3475]
I came here, because i'm starting to lose my mind:
i do vj stuff witb OBS studio, tickin' on and off sources and some I -normally- keep running when VJ'ing, but when I'm done I [x] CLOSE-when-inactive .. however, the videos keep running in the background. Even when the scene is empty and a completely different [and empty] scene is selected, OBS still seems to run them, since Resource Monitor clearly shows these videos running. Causing lag in encoding of other stuff and gobbles up 20% of resources on an idling obs-studio.
I went to #windows-support and had my log checked multiple times, looked into my obs sources; removed most that i did not need, or had removed but still had the source in my scene, re-ticked all media-sources that had video with [x] close-when-inactive but as soon i check again, the videos are still running. Closed obs-studio, started back up with the [x] close-when-inactive settings, but they are still running.

It's driving me nuts: an idling doing nothing obs, running 5-6 videos in the background for no reason when [x] close-when-inactive and I've no ideas left what to do. 

I think this is a bug and I can repeativly replicate this.

## Platform

Operating system and version:

Windows 10 Pro - 64bit - 2004
OS Build 19041.508

OBS Studio version:

OBS Studio 25.0.8 (64 bit version - Windows) 

## Expected Behavior

CPU Utilization should be really low and no videos should be playing in the background according to Resource Monitor.

## Current Behavior

The Media Sources KEEP playing in the background, when [x] close-file-when-inactive is ticked. 
In my case videos, which are quite heavy on resources. Even when set to a new scene, completly empty, Resource Monitor will still show the videos being played back (hdd activity) and causing OBS to increase CPU utilization.
![image](https://user-images.githubusercontent.com/26382274/93268351-507d1900-f7ad-11ea-9349-a132c8e26345.png)

https://obsproject.com/logs/hWrXSsC8f2e9mI5C

## Steps to Reproduce

1. Start up OBS Studio.
2. Wait until everything is loaded. 
3. CPU utilization @ +- 20% when idling
4. Resource Monitor shows the videos that are set in a scene as source to be [x] close-when-inactive to be actively running.

![image](https://user-images.githubusercontent.com/26382274/93267805-7b1aa200-f7ac-11ea-840c-f91f12cf27e8.png)

## Additional information

in short-short: when [x] close-when-inactive .. it doesn't close and remains active even if not shown on screen or turned on in the scene in OBS

**Comment by Fenrirthviti** [comment]
I am not able to replicate this behavior. Video files on my end are not still playing in the background when they are not active, regardless of the checkbox. I think there might be a misunderstanding somewhere.

Can you please test with a completely new scene collection and validate that this behavior is still happening? Make a new scene collection (make sure this step is a COLLECTION, not just a new scene), add only a single media source, set the file to close when inactive, and then create a new scene and switch to it, and let me know if this is still occurring.

**Comment by MACKerMD** [comment]
Done as decribed. Then it does not happen. Nor does it show up in resources. 
-then- is seems really inactive.

**Comment by energizerfellow** [comment]
In addition to the test that @Fenrirthviti mentioned, if the issue is still there, might be StreamElements' OBS.Live breaking the media playback in some way.

Note, however, that you simply can't uninstall OBS.Live and have OBS Studio fully functional afterwards due to some components that get replaced.  You'll need to do the following, in order:

- Uninstall OBS.Live
- Uninstall OBS Studio (make sure that "User Settings" is not selected)
- Reinstall OBS Studio

**Comment by Fenrirthviti** [comment]
I'm going to close this issue which is confirmed to not be a bug within OBS, but something specific to your setup or scene collection. Please use the Discord of Forums if the above steps do not fix the issue, as we do not want to use the Issues page for general support.

https://obsproject.com/help
